conf/nginx/st2.conf

```diff
@@ -40,6 +40,10 @@ server {
 
   add_header              Front-End-Https on;
   add_header              X-Content-Type-Options nosniff;
+  add_header              X-Frame-Options DENY always;
+  add_header              Strict-Transport-Security "max-age=3153600;includeSubDomains";
+  add_header              Cache-Control "no-store; no-cache; max-age=0; must-revalidate;";
```

Comment:
```suggestion
```
@ashwini-orchestral can you please remove the `Cache-Control` headers per team feedback?